examples/ZMQPP/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.2)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-ZMQPP)
+
+hunter_add_package(ZMQPP)
+
+find_package(ZMQPP CONFIG REQUIRED)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
I have a problem. I added the following lines to the example CMakelists.txt, before `hunter_add_package` command:

```cmake
if(NOT MSVC)
  string(COMPARE EQUAL "${CMAKE_TOOLCHAIN_FILE}" "" no_toolchain)
  if(no_toolchain)
    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
    set(CMAKE_CXX_STANDARD 11)
  endif()
endif()
```
I debugged with messages that we enter in the second condition and set the variables but looks like it has no effect. I got the c++11 missing error message from the ZMQPP's CMakelists.txt. I tried to put this variable setting before including common.cmake but has no effect as well. What is wrong?